CMakeLists.txt

```diff
@@ -147,7 +147,7 @@ set( AMDGPU_TARGETS "${GPU_TARGETS}" CACHE STRING "List of GPUs (gfx targets) to
 
 option( BUILD_WITH_HIPBLASLT "Build with HipBLASLt" ON )
 set( hipblaslt_path "/opt/rocm" CACHE PATH "Use local HipBLASLt directory" )
-set( HIPBLASLT_VERSION 0.13 CACHE STRING "The version of HipBLASLt to be used" )
+set( HIPBLASLT_VERSION 0.15 CACHE STRING "The version of HipBLASLt to be used" )
```

Comment:
Did you get a version already, maybe can keep at 0.13 unless they added something new we need?

Reply:
The hipBLASLt develop branch updated from version 0.13 to version 0.15 on April 1 with commit https://github.com/ROCm/hipBLASLt/commit/a21cb83325738cce62c676809595a6ef291fa412